### PR TITLE
Add missing config keys

### PR DESCRIPTION
**Pull request by jpereira**
It was reported by scripts/build/missing-raddb-mod-conf.sh

### CODE REVIEW

File: raddb/debug.conf [hunk 1]
```diff
@@ -14,6 +14,31 @@
...
+
+#
+#  talloc_memory_report:: Print a summary report of all memory used.
+#
+#talloc_memory_report = false
+
+#
+#  talloc_pool_size:: A talloc pool is a pure optimization for specific situations.
+#
+#talloc_pool_size = 8192
```
Comment (arr2036) [inline raddb/debug.conf]: These are all developer only options and should not be documented in the public config (well except debug_level).  If you want to add a "do nothing" flag (just expands to 0) like FR_TYPE_HIDDEN to the CONF_PARSER entries, and then ignore lines containing that, that's probably the best way to manage this.

File: raddb/mods-available/eap [hunk 2]
```diff
@@ -410,6 +420,12 @@ eap {
 		#
 		dh_file = ${certdir}/dh
 
+		#
+		#  disable_single_dh_use:: Always using it has an impact on the computer time needed
+		#  during negotiation, but it is not very large.
```
Comment (arr2036) [inline raddb/mods-available/eap]: That's not really a meaningful comment...

File: raddb/mods-available/eap_inner [hunk 3]
```diff
@@ -105,11 +110,22 @@ eap inner-eap {
...
+#		verify_depth = 0
+
 		#
 		#  dh_file::
 		#
 		dh_file = ${certdir}/dh
 
+		#
+		#  disable_single_dh_use:: Always using it has an impact on the computer time needed
+		#  during negotiation, but it is not very large.
```
Comment (arr2036) [inline raddb/mods-available/eap_inner]: Same here.  If you're going to document it, figure out what it actually does

File: raddb/radiusd.conf.in [hunk 4]
```diff
@@ -286,6 +286,11 @@ log {
 	#
 #	timestamp = no
 
+	#
+	#  use_utc:: Enable to use UTC time.
```
Comment (arr2036) [inline raddb/radiusd.conf.in]: Should be "Print all timestamps in log messages as UTC"

Comment (arr2036) [reply raddb/radiusd.conf.in]: @alandekok if you get a moment, sure would be nice to make this a global toggle for printing value boxes too.  See telkom defect list :(

File: raddb/mods-available/eap (hunk 2 above)
Comment (jpereira) [reply raddb/mods-available/eap]: @arr2036 updated

**Comment by jpereira** [comment]
as soon as you get a chance @arr2036

File: raddb/clients.conf [hunk 5]
```diff
@@ -136,6 +136,21 @@ client localhost {
...
+	#
+#	virtual_server = default
+
+	#
+	#  response_window:: How long the client has to respond.
+	#
+#	response_window = 0
+
+	#
+	#  track_connections:: Allow connected sockets to be set on a per-client basis.
```
Comment (arr2036) [inline raddb/clients.conf]: I don't understand what this means, and if I don't, people reading the docs probably won't either.  Could you seek clarification as to what this config item does from @alandekok

File: raddb/mods-available/couchbase [hunk 6]
```diff
@@ -238,6 +238,23 @@ couchbase {
 		#
 		connect_timeout = 3.0
 
+		#
+		#  held_trigger_min:: If a connection is held for less than the specified
+		#  period, fire a trigger.
```
Comment (arr2036) [inline raddb/mods-available/couchbase]: What does "held" mean in this context.  Can all three config items be updated with less ambiguous terms.

File: raddb/mods-available/eap [hunk 7]
```diff
@@ -250,6 +255,11 @@ eap {
 			#
 			ca_file = ${certdir}/rsa/ca.pem
 
+			#
+			#  verify_depth:: Maximum number of certificates we can traverse.
```
Comment (arr2036) [inline raddb/mods-available/eap]: "Maximum number of certificates we traverse when attempting to validate the certificate chain."

English is hard :(

File: raddb/mods-available/redis [hunk 8]
```diff
@@ -94,6 +94,23 @@ redis {
...
+#		held_trigger_min = 0
+
+		#
+		#  held_trigger_max:: If a connection is held for longer than the specified
+		#  period, fire a trigger.
+		#
+#		held_trigger_max = 0.5
+
+		#
+		#  max_pending:: Max number of pending connections to allow.
```
Comment (arr2036) [inline raddb/mods-available/redis]: What does pending open mean here, is it semi-open, or connecting connections?

File: raddb/radiusd.conf.in [hunk 9]
```diff
@@ -272,6 +272,11 @@ log {
 	#
 	colourise = yes
 
+	#
+	#  line_number:: Print src file and line number in log messages.
+	#
+#	line_number = no
```
Comment (arr2036) [inline raddb/radiusd.conf.in]: Developer only option

**Pull request closed without merging**